membership-attribute-service/app/components/TouchpointBackends.scala

```diff
@@ -22,7 +22,7 @@ class TouchpointBackends(
     contactRepositoryOverride: Option[ContactRepository] = None,
     subscriptionServiceOverride: Option[SubscriptionService[Future]] = None,
     zuoraRestServiceOverride: Option[ZuoraRestService] = None,
-    catalogServiceOverride: Option[Future[CatalogMap]] = None,
+    catalogServiceOverride: Option[Future[Catalog]] = None,
```

Comment:
Catalog is now an object containing the product ids as well (to make it easier to pass around)